<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>key_change</class>
 <widget class="QMainWindow" name="key_change">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>密钥交换</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_key_change_get_shared_key">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>440</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>生成共享密钥并保存</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icons/icons/-_另存为.png</normaloff>:/icons/icons/-_另存为.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>用户A</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>20</width>
      <height>391</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>65</y>
      <width>72</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>用户B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_key_change_cho_pri_file_1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>选择私钥文件</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icons/icons/-_打开项目.png</normaloff>:/icons/icons/-_打开项目.png</iconset>
    </property>
   </widget>
   <widget class="CustomLineEdit" name="lineEdit_key_change_dir_pri_file_1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_key_change_cho_pub_file_1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>340</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>选择公钥文件</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icons/icons/-_打开项目.png</normaloff>:/icons/icons/-_打开项目.png</iconset>
    </property>
   </widget>
   <widget class="CustomLineEdit" name="lineEdit_key_change_dir_pub_file_1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_key_change_cho_pri_file_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>190</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>选择私钥文件</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icons/icons/-_打开项目.png</normaloff>:/icons/icons/-_打开项目.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_key_change_cho_pub_file_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>340</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>选择公钥文件</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icons/icons/-_打开项目.png</normaloff>:/icons/icons/-_打开项目.png</iconset>
    </property>
   </widget>
   <widget class="CustomLineEdit" name="lineEdit_key_change_dir_pub_file_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>270</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="CustomLineEdit" name="lineEdit_key_change_dir_pri_file_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>120</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">customlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
